aim/sdk/objects/image.py

```diff
@@ -101,29 +106,52 @@ def json(self):
             'height': self.height,
         }
 
-    def _from_pil_image(self, pil_image: PILImage.Image):
+    def _from_pil_image(self, pil_image: PILImage.Image, compress=False):
         assert isinstance(pil_image, PILImage.Image)
-        image_bytes_p = BytesIO()
-        pil_image.save(image_bytes_p, format='png')
+        img_container = BytesIO()
+
+        if compress:
+            if pil_image.mode == 'RGBA' or isinstance(pil_image.info.get('transparency'), bytes):
+                # Remove transparency
+                alpha = pil_image.convert('RGBA').split()[-1]  # Get only alpha
+                background = PILImage.new('RGBA', pil_image.size, (255, 255, 255, 255))
+                background.paste(pil_image, mask=alpha)
+                pil_image = background.convert('RGB')
+
+            params = {
+                'format': 'jpeg',
+                'optimize': True,
+                'quality': 80
```

Comment:
@devfox-se how much compression we get with this values?

Reply:
On the second thought it's better to let the user define format and the quality